### PR TITLE
Doc updates for Rack 20160928105531

### PR DESCRIPTION
**Pull request by mattmanning**
## Release Playbook
- [x] Rebase against master
- [x] Code review
- [ ] Merge into master
- [ ] Verify changes at http://site-staging.convox.com
- [ ] Promote release on `site-production` in the `convox/production` Rack

### CODE REVIEW

**Comment by mattmanning** [comment]
[Doc updates for Rack 20160928105531](https://trello.com/c/TXgFTf2R/779-doc-updates-for-rack-20160928105531)

File: _deployment/builds.md [hunk 1]
```diff
@@ -27,11 +27,23 @@ The newly created release will not be promoted (made active) until you run `conv
 
 If you'd like to build your app and promote the release in a single step, you can run `convox deploy` rather than `convox build`.
 
+### Building From a Git Repository
+
+You can tell convox to build from a remote git repository rather than shipping your local files. To do so, pass the URL (with the .git extension) as an argument to `convox build`. For example:
```
Comment (ianmalott) [inline _deployment/builds.md]: convox -> Convox

File: _management/one-off-processes.md [hunk 2]
```diff
@@ -19,6 +19,20 @@ $ convox run web bin/migrate
 Migrating database... Done
 ```
 
+By default, `convox run` processes are run as "attached." This means that the output from your terminal is piped to the input of the process and vice versa.
```
Comment (ianmalott) [inline _management/one-off-processes.md]: >  This means that the output from your terminal is piped to the input of the process and vice versa.

Use of output/input here might be confusing. What about something like this?

"This means the user can continue to interact with the process via the terminal (stdin and stdout)."

**Comment by mattmanning** [comment]
@ianmalott Good suggestions. Changed both.

**Pull request merged**